data_uploader/app/admin/data_uploader/use_case/upload_datafile.rb

```diff
@@ -0,0 +1,44 @@
+module DataUploader
+  module UseCase
+    class UploadDatafile < DataUploader::UseCase::Base
+      def call(params, callbacks)
+        validate_if_file_chosen params
```

Comment:
just realized that the validations crash the app; we need to pass `callbacks` to the validation methods. Both to `validate_if_file_chosen ` and `validate_content_type_against_csv`. Sorry for that, I must've missed that